scene/gui/texture_button.cpp

```diff
@@ -178,13 +182,16 @@ void TextureButton::_notification(int p_what) {
 				texdraw = focused;
 			}
 
-			if (texdraw.is_valid()) {
-				size = texdraw->get_size();
-				_texture_region = Rect2(Point2(), texdraw->get_size());
+			if (texdraw.is_valid() || texdraw_bitmap.is_valid()) {
+				Size2 texdraw_size = texdraw.is_valid() ? texdraw->get_size() : Size2(texdraw_bitmap->get_size());
+				float texdraw_height = texdraw.is_valid() ? texdraw->get_height() : texdraw_size.height;
+				float texdraw_width = texdraw.is_valid() ? texdraw->get_width() : texdraw_size.width;
```

Comment:
These variables don't make sense. You can just always use `texdraw_size.height`.

Reply:
true, although not that apparent, applied suggestion